src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -226,13 +225,33 @@ bool AlertManeuverRequest::IsWhiteSpaceExist() {
     smart_objects::SmartArray::const_iterator it_tc_end = tc_array->end();
 
     for (; it_tc != it_tc_end; ++it_tc) {
-      str = (*it_tc)[strings::text].asCharArray();
+      const char* str = (*it_tc)[strings::text].asCharArray();
       if (strlen(str) && !CheckSyntax(str)) {
         LOG4CXX_ERROR(logger_, "Invalid tts_chunks syntax check failed");
         return true;
       }
     }
   }
+  if ((*message_)[strings::msg_params].keyExists(strings::soft_buttons)) {
+    DCHECK_OR_RETURN(
```

Comment:
@AKalinich-Luxoft smart object schema validation is responsible for correct type of this field. It it is wrong - this requests should not be executed. Ant there it is fatal error